src/test/resources/canonicalDDO.json

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "did:sora:uuid:caab4570-5f3f-4050-8d61-15306dea4bcf",
+  "publicKey": [
+    {
+      "id": "did:sora:uuid:caab4570-5f3f-4050-8d61-15306dea4bcf#keys-1",
+      "type": "Ed25519Sha3VerificationKey",
+      "owner": "did:sora:uuid:caab4570-5f3f-4050-8d61-15306dea4bcf",
+      "publicKeyValue": "02B97C30DE767F084CE3080168EE293053BA33B235D7116A3263D29F1450936B71"
+    },
+    {
+      "id": "did:sora:uuid:caab4570-5f3f-4050-8d61-15306dea4bcf#keys-2",
+      "type": "Ed25519Sha3VerificationKey",
+      "owner": "did:sora:uuid:caab4570-5f3f-4050-8d61-15306dea4bcf",
+      "publicKeyValue": "6C0816523644DD538B756F88D5043C311764D44221926E91A189FDA20B96D491"
+    }
+  ],
+  "authentication": [
+    {
+      "type": "Ed25519Sha3Authentication",
+      "publicKey": "did:sora:uuid:caab4570-5f3f-4050-8d61-15306dea4bcf#keys-2",
+      "owner": null,
```

Comment:
Please disable inclusion of null values.

https://github.com/soramitsu/1code-java/blob/master/src/main/java/jp/co/soramitsu/jackson/OneCodeMapper.java#L30